docs-ref-services/cognitive-services.md

```diff
@@ -13,31 +13,34 @@ ms.devlang: nodejs
 ms.service: Cognitive Services
 ---
 
-# Azure Cognitive Services packages for Node.js
+# Azure Cognitive Services Modules for Node.js
 
 ## Overview
+
 Azure Cognitive Services is a set of APIs, SDKs, and services available to developers to make their applications more intelligent, engaging and discoverable. Microsoft Cognitive Services expands on Microsoft’s evolving portfolio of machine learning APIs and enables developers to easily add intelligent features – such as emotion and video detection; facial, speech and vision recognition; and speech and language understanding – into their applications. Our vision is for more personal computing experiences and enhanced productivity aided by systems that increasingly can see, hear, speak, understand and even begin to reason.
 
 ## Management Package
 
-### Install npm package
+### Install the npm module
 
-Install the Azure Cognitive Services module for Node.js
+Install the Azure Cognitive Services npm module
 
 ```bash
 npm install azure-arm-cognitiveservices
 ```
 
 ### Example
+
+This example lists all cognitive service accounts.
+
 ```javascript
 const msRestAzure = require('ms-rest-azure');
 const cognitiveServicesManagementClient = require('azure-arm-cognitiveservices');
 
 const subscriptionId = 'your-subscription-id';
-let cognitiveServicesClient;
 
 msRestAzure.interactiveLogin().then(credentials => {
-  cognitiveServicesClient = new cognitiveServicesManagementClient(credentials, subscriptionId);
+  const cognitiveServicesClient = new cognitiveServicesManagementClient(credentials, subscriptionId);
   cognitiveServicesClient.cognitiveServicesAccounts
     .list()
     .then(cognitiveServicesAccounts =>

```

Comment:
In the other (at least most) code examples we've been returning promises and chaining them instead of nesting them, I think we should do that here as well.